config/metadata-processor/src/main/java/io/helidon/config/metadata/processor/ConfigMetadataHandler.java

```diff
@@ -109,7 +109,10 @@ synchronized void init(ProcessingEnvironment processingEnv) {
 
         // get the types
         configuredElement = elementUtils.getTypeElement(CONFIGURED_CLASS);
-        builderType = elementUtils.getTypeElement("io.helidon.common.Builder").asType();
+        TypeElement builderTypeElement = elementUtils.getTypeElement("io.helidon.common.Builder");
```

Comment:
Any element annotated with metadata must have `io.helidon.common.Builder` on classpath, as otherwise the annotation processor does not make sense.

Reply:
Eclipse IDE does not work very well with Maven (this happens more when there are plugins doing particular things). Possibly if you tune the module-path a bit you could end up fixing this issue.

But then, you are making it more complicated for any dev that is using Eclipse IDE and wants to contribute in Helidon.

This change does not have side effects, and it will help users to contribute in Helidon. For this reason I think we should include it.

We can add a WARNING if you want.